src/database/DbLogin.java

```diff
@@ -0,0 +1,34 @@
+package database;
+import java.security.KeyStore.Entry;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+public class DbLogin {
+	
+	
+	Map <String, String>loginPassword=new HashMap<>();
+	
+	public DbLogin(){
+			loginPassword.put("asalinas","1234");
+			loginPassword.put("bsalinas","5678");
+			loginPassword.put("csalinas","9876");
+			loginPassword.put("dsalinas","5432");
+			loginPassword.put("esalinas","1234");
+			loginPassword.put("fsalinas","5678");
+			loginPassword.put("gsalinas","9876");
+			loginPassword.put("hsalinas","5432");
+			loginPassword.put("isalinas","1234");
+			loginPassword.put("jsalinas","5678");	
+
+	}
+	public boolean esIgual(String login, String constrasena) {
+		if(loginPassword.get(login).equals(constrasena)){
```

Comment:
When the user does not exist this sentence will fail with a nullpointer exception, please review why. An example of an user that does not exist is `ksalinas`.